pom.xml

```diff
@@ -472,6 +465,12 @@ THE SOFTWARE.
       <artifactId>promoted-builds</artifactId>
       <version>2.23</version>
       <scope>test</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>org.jenkins-ci.main</groupId>
+          <artifactId>maven-plugin</artifactId>
+        </exclusion>
+      </exclusions>
```

Comment:
Strange that you can actually depend on an old version of a plugin; surprised that Maven allows this.  Glad that this is being removed.